### ISSUE TITLE
Interface to handle twilio replies

### ISSUE THREAD

**Issue by willgearty** [learning-unlimited/ESP-Website#2575]
Sometimes people reply to Twilio texts, and it would be nice to be able to receive those replies and then possibly respond to them to the proper phone numbers. A quick fix is possible through Twilio forwarding, but the message ends up looking like this: 

From: +1XXXXXXXXXX
Body: Running late but I'll be there on time!

This comes from the Twilio phone number, which makes it annoying to reply directly to these messages (you need to copy the number and make a new message).

Alternatively, it would be cool to have an interface on the website to interact with the Twilio account via the API.

Requested by Stanford

**Comment by benjaminjkraft** [comment]
Hmm, does twilio have a web UI to do this?  It seems like a lot of work to build our own, unless it has a significant value add.

**Comment by willgearty** [comment]
I will defer to @milescalabresi

**Comment by milescalabresi** [comment]
I don't think it comes with one on the site, but there are existing templates and tools that would significantly reduce the work required to build the UI. It looks like Twilio is coming out (future tense) with its own building blocks <https://www.twilio.com/frame>, and you can pay for drag-and-drop style UI developer tools <https://www.twilio.com/showcase/onereach>. There are also blog posts (e.g. https://www.twilio.com/blog/2017/06/complete-chat-application-javascript.html) that walk you through the basics of a UI build.

**tl;dr** no, but there are tools that make it so you don't have to create one from scratch

**Comment by Dhirenderchoudhary** [comment]
@willgearty i can work on it if it’s not fixed yet

**Comment by kkbrum** [comment]
This one requires having access to some of our Twilio info, so we'll hold off on assigning it for now.
